src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/src/types/index.ts

```diff
@@ -14,9 +16,8 @@ import { IOrgsState } from '../shared/resources/orgs/orgsReducers';
 import { IPartyState } from '../shared/resources/party/partyReducers';
 import { IProcessState } from '../shared/resources/process/processReducer';
 import { IProfileState } from '../shared/resources/profile/profileReducers';
-import {IQueueState} from '../shared/resources/queue/queueReducer';
+import { IQueueState } from '../shared/resources/queue/queueReducer';
 import { ITextResourcesState } from '../shared/resources/textResources/textResourcesReducer';
-import { IFormRuleState } from 'src/features/form/rules/rulesReducer';
 
```

Comment:
Can I be really picky here, and say that since we are doing a cleanup, can we order the interfaces and enums alphabetically? 😇  I think it makes the whole file more readable, and it's easy to know where to add a new interface later.
To specify: I don't mean within the defined interfaces themselves at this point, but that the interface definitions in this file are ordered alphabetically 😄